pandas/core/series.py

```diff
@@ -1231,6 +1232,14 @@ def mode(self):
         # TODO: Add option for bins like value_counts()
         return algos.mode(self)
 
+    @Appender(base._shared_docs['unique'] % _shared_doc_kwargs)
+    def unique(self):
+        result = super(Series, self).unique()
```

Comment:
Because ``Series.unique`` definition is required for docstring after all, performing ``Series`` specific process here looks clearer.